netbox/utilities/views.py

```diff
@@ -144,7 +144,7 @@ def get(self, request):
             table.columns.show('pk')
 
         # Construct queryset for tags list
-        if hasattr(model, 'tags'):
+        if hasattr(model, 'tags') and type(model.tags).__name__ is not 'ManyToManyDescriptor':
```

Comment:
How about:

```
from django.db.models.manager import Manager
...
if hasattr(model, 'tags') and issubclass(model.tags, Manager):
```

That's probably conveys our intention a little better. Admittedly, just using `hasattr` was lazy on my part.